### ISSUE TITLE
Incorrect webpage link display on hover

### ISSUE THREAD

**Issue by prachiagrawal269** [Tonejs/Tone.js#218]
On the "Phaser" page under the "Effect" menu the webpage link that gets displayed on hovering over the term "Wikipedia" in line 2 of the contents is incorrect. There is a closing bracket missing from the displayed link.
(Observe the bottom left corner of the webpage)
![imageedit_1_6382164269](https://cloud.githubusercontent.com/assets/9863912/24161712/2e9e5952-0e8b-11e7-8a48-a75005ce25d8.gif)

Current link: https://en.wikipedia.org/wiki/Phaser_(effect
Correct link: https://en.wikipedia.org/wiki/Phaser_(effect)

**Comment by tambien** [comment]
@prachiagrawal269 Good catch. Thanks! Seems like there's a problem with the way i'm parsing markdown links when i generate the docs. I'll take care of that soon.

**Comment by prachiagrawal269** [comment]
@tambien I tried using markdown livedemo (referred link: http://markdown-here.com/livedemo.html)
and this is what I found:
![screenshot from 2017-03-22 01 19 01](https://cloud.githubusercontent.com/assets/9863912/24167669/8e8dde7a-0e9d-11e7-8b0a-f059a30870d8.png)


I think the problem is with the way the markdown translation works. May be during the translation the webpage link that is being considered is until the occurence of the first closing round bracket (instead of matching valid parenthesis).
One hack to this problem could be the following:
![screenshot from 2017-03-22 01 13 46](https://cloud.githubusercontent.com/assets/9863912/24167623/67e33f22-0e9d-11e7-99a4-aeb323eaf758.png)

Basically replacing the round brackets with square brackets along with giving index to the referred webpage link ([1] in this case) and inserting that index in the second set of squared brackets might work.

**Comment by tambien** [comment]
New docs and [the link](https://tonejs.github.io/docs/r11/Phaser) issue is fixed in the latest. It had to do with the way that i was processing markdown which should be corrected now. 

![image](https://user-images.githubusercontent.com/1323402/30516469-464b253e-9b0e-11e7-9867-947242346800.png)
